### DISCUSSION TITLE
Feature: Matroska output

### DISCUSSION THREAD

**Discussion by promovicz** [asciinema/discussions#80] [Ideas]
### Use cases

- Recording terminal sessions with audio for instructional purposes.
- Possibly live streaming of terminal sessions with cut-n-paste support as well as a keystroke log.

### Implementation

Could be done by supporting asciicasts in matroska streams, which could then be piped into FFMPEG or similar to add video and audio.

**Comment by zurgeg** [comment]
I know this is issue is pretty old (over a year old) but for live streaming I think we could stream the keystroke data over a TCP connection, which would be pretty straight forward (albeit uncompressed, even though part of asciicast's whole thing is it doesn't need compression)
